code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -179,9 +179,9 @@
 
 /datum/chemical_reaction/emp_pulse/on_reaction(datum/reagents/holder, created_volume)
 	var/location = get_turf(holder.my_atom)
-	// 100 created volume = 4 heavy range & 7 light range. A few tiles smaller than traitor EMP grandes.
-	// 200 created volume = 8 heavy range & 14 light range. 4 tiles larger than traitor EMP grenades.
-	empulse(location, round(created_volume / 12), round(created_volume / 7), 1)
+	// 100 created volume = 8 heavy range & 14 light range. A few tiles smaller than traitor EMP grandes.
+	// 200 created volume = 10 heavy range & 24 light range. 4 tiles larger than traitor EMP grenades.
+	empulse(location, min(round(created_volume / 12), 10), min(round(created_volume / 7), 1), 24) // Don't emp the station mr chemist, I like my time dialiation how it is
```

Comment:
I think your parenthesis are in the wrong order here? Currently that min is restricting the light range to 1 and you're putting a 24 in whatever the fourth arg of `empulse` is.

Reply:
ahhhh you're right, my mistake